pywinauto/timings.py

```diff
@@ -111,6 +111,9 @@ class TimeConfig(object):
         'app_start_timeout' : 10.,
         'app_start_retry' : .90,
 
+        'app_connect_timeout': 5.,

```

Comment:
Maybe it's better to use this value? In case when `timeout=None` by default. It could allow us manipulating it as all other timings.